symfony/framework-bundle/3.3/Makefile

```diff
@@ -6,14 +6,22 @@ cache-warmup: cache-clear
 	@test -f bin/console && bin/console cache:warmup || echo "cannot warmup the cache (needs symfony/console)"
 .PHONY: cache-warmup
 
+CONSOLE=bin/console
+$(CONSOLE):
+	@exit
+
+serve_as_sf: $(CONSOLE)
+	@$(CONSOLE)|grep server:start > /dev/null
+	@$(CONSOLE) server:start || exit 0
```

Comment:
looks wrong to me. If there is a failure in the command, we want to report it in the main exit code, to let the external system know that it failed